### DISCUSSION TITLE
How to check if message is readed

### DISCUSSION THREAD

**Discussion by Kliton** [gotd/td#697] [General]
Hi, i've sent a message to a InputPeer ( user peer id ).

I got back a update with the message Id. How i can check ( later on ) if the message has been readed?

**Comment by ghost** [comment]
Telegram sends an update about message read
- Update for channel: [`updateReadChannelOutbox`](https://core.telegram.org/constructor/updateReadChannelOutbox)
- Update for PM (direct messages) and legacy groups: [`updateReadHistoryOutbox`](https://core.telegram.org/constructor/updateReadHistoryOutbox)
- Multiple updates for PM (direct messages) and legacy groups: [`updateReadMessagesContents`](https://core.telegram.org/constructor/updateReadMessagesContents)
